plugins/inputs/amqp_consumer/amqp_consumer.go

```diff
@@ -98,10 +99,13 @@ func (a *AMQPConsumer) SampleConfig() string {
   # exchange_arguments = { }
   # exchange_arguments = {"hash_propery" = "timestamp"}
 
-  ## AMQP queue name
+  ## AMQP queue name.
   queue = "telegraf"
 
-  ## Binding Key
+  ## AMQP queue durability.
+  queue_durability = true
```

Comment:
Can you make this a string that can be either "durable" or "transient", defaulting to "durable".  This will just match up with the exchange_durability option a bit better.

Reply:
@danielnelson Good idea, just fixed that